src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -3988,10 +4787,10 @@ await RunFormattingTestAsync(
                         private void M()
                         {
                             object entries = new()
-                                {
-                                    First = 1,
-                                    Second = 2
-                                };
+                            {
+                                First = 1,
+                                Second = 2
+                            };
                         }
```

Comment:
These changes are a result of the new engine fixing https://github.com/dotnet/razor/issues/7933